releasetasks/templates/release_graph.yml.tmpl

```diff
@@ -65,6 +65,19 @@ tasks:
     {% endif %}
 
     {% if verifyConfigs %}
+        {% macro updateVerify_task(platform) %}
+            {% if platform in ["linux", "linux64"] %}
```

Comment:
I do the for loop over platforms below, this is the macro that takes an argument....  We'll need a different task for the buildbot-based update verifies, (e.g. windows and mac) since they can't run on linux.